### ISSUE TITLE
Handler for tag 1 does not exist

### ISSUE THREAD

**Issue by foyarash** [kmagiera/react-native-gesture-handler#182]
Hi,

I got this error using PanGestureHandler.

My code looks like:

```
<PanGestureHandler enabled maxPointers={1} onGestureEvent={this.handleEvent}>
     <Animated.View { ...someStuff }>
     </Animated.View>
</PanGestureHandler>
```

Am I missing something ?

**Comment by osdnk** [comment]
Hi, @foyarash 
Your code seems to be fine (however `enabled` is not required, it's enabled by default, but it's not a matter of this issue). Would you mind giving me some snippet for reproduce this issue?

**Comment by foyarash** [comment]
@osdnk In my case, I have a FlatList on which each item has a PanGestureHandler.

The problem seems to occur when I have my item component in a different file from the FlatList.

The following works correctly: https://pastebin.com/XVX1bnX4

But having the Item component in a separate file breaks, and the error appears

**Comment by osdnk** [comment]
@foyarash 
https://github.com/osdnk/flat-list-react-native-gesture-handler-sample
I made a project to see your issue and still cannot reproduce it

**Comment by osdnk** [comment]
This project works only for iOS. Do you face this issue on both platforms?

**Comment by foyarash** [comment]
Tested your project on both iOS and android.

android does not work at all, but doesnt display any red box
iOS works when the Text tag has a style

**Comment by osdnk** [comment]
Android does not work, because I didn't configure it 😃 .  
Well, it's quite interesting issue on iOS. I'll take a look

**Comment by foyarash** [comment]
@osdnk another thing i noticed is that if i set a margin <= 5 on the Text, it crashes

**Comment by Saranchenkov** [comment]
I have the same issue (on Android). I have investigated this problem and found, that createGestureHandler method have invoked after attachGestureHandler method. Other words, library try to attach gesture handler that doesn't exist yet.
It is happening because of "setImmediate" method in componentDidMount.
So when app started, componentDidMount was invoked, and then componentDidUpdate was invoked immediately (because of redux action in componentDidMount). When i insert console.log into GestureHandler.js file, i see the following sequence of methods invocation:
1) componentDidMount
2) componentDidUpdate and attaching of gesture handler **(that doesn't exist yet)** to view
3) setImmediate inside componentDidMount, creation of gesture handler

When i remove redux actions from componentDidMount, i haven't error

**Comment by foyarash** [comment]
@Saranchenkov thank you for pointing this out, i was able to fix my issue thanks to this !

My issue was that i launched an animation on componentDidMount, i just removed this animation and it correctly works now

**Comment by foyarash** [comment]
@osdnk  Thank you for being so reactive

**Comment by kmagiera** [comment]
@foyarash thanks for reporting and responding so quickly 👏 . Just published 1.0.1 that includes this fix: https://github.com/kmagiera/react-native-gesture-handler/releases/tag/1.0.1

Let us know if it fixes your problem so that we can close the issue.

**Comment by foyarash** [comment]
@kmagiera can confirm it works, thank you!
